contracts/staking/TribalChief.sol

```diff
@@ -94,12 +95,17 @@ contract TribalChief is CoreRef, ReentrancyGuard {
     /// @dev Total allocation points. Must be the sum of all allocation points in all pools.
     uint256 public totalAllocPoint;
 
+
+    /// @notice these variables cannot be initialized with a constant value 
+    /// because of the way that upgradeable contracts work, their initial write
+    // to storage doesn't happen as the logic contract doesn't run the constructor
+
     /// the amount of tribe distributed per block
-    uint256 private tribalChiefTribePerBlock = 1e20;
+    uint256 private tribalChiefTribePerBlock;
     /// variable has been made constant to cut down on gas costs
-    uint256 private constant ACC_TRIBE_PRECISION = 1e23;
```

Comment:
Why are we getting rid of constants? They should be hardcoded into the bytecode, and therefore, usable on a proxy pattern.

I admittedly am not an expert on ERC 1967 but since it uses `delegatecall`, all the bytecode positions should be available in a certain execution frame. The constants are built at compile time, not deploy time.

Reply:
I'm pretty sure this is the case as well so we can keep constants